### PR TITLE
Add variable for VPA to be enabled

### PR DESCRIPTION
**Pull request by transient1**

### CODE REVIEW

File: vpc-native/inputs.tf [hunk 1]
```diff
@@ -73,3 +73,7 @@ variable "logging_service" {
   description = "The logging service to write logs to"
   default     = "logging.googleapis.com/kubernetes"
 }
+
+variable "vpa_enabled" {
+  description = "A boolean to enable VPA for the cluster"}
+  default = false
```
Comment (lucasreed) [inline vpc-native/inputs.tf:79]:
```suggestion

variable "vpa_enabled" {
  description = "A boolean to enable VPA for the cluster"
  default = false
}
```

**Review by lucasreed — CHANGES_REQUESTED** [review]

**Review by sudermanjr — CHANGES_REQUESTED** [review]
Just adding the variable to the inputs will not be sufficient for what you're trying to do. You'll need to add the vpa configuration block to the gke cluster module in main.tf and consume this variable there.

File: vpc-native/main.tf [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-  monitoring_service = var.monitoring_service
-  logging_service    = var.logging_service
+  name                     = var.name
+  location                 = var.region
+  min_master_version       = var.kubernetes_version
+  network                  = var.network_name
+  subnetwork               = var.nodes_subnetwork_name
+  monitoring_service       = var.monitoring_service
+  logging_service          = var.logging_service
+  vertical_pod_autoscaling = var.vpa_enabled
```
Comment (sudermanjr) [inline vpc-native/main.tf:9]: I think this needs to be a block, not just a true/false. 

Documentation here:
https://www.terraform.io/docs/providers/google/r/container_cluster.html

Also, is this still a beta provider feature? If so, we might have to do something to handle that.

**Review by sudermanjr — CHANGES_REQUESTED** [review]

File: vpc-native/main.tf (hunk 2 above)
Comment (transient1) [reply vpc-native/main.tf]: I don't see that it's beta in the description. Some other parameters specify `(Optional, Beta)`.

Comment (sudermanjr) [reply vpc-native/main.tf]: Screen Shot 2020-10-22 at 1 24 12 PM

Comment (sudermanjr) [reply vpc-native/main.tf]: Screen Shot 2020-10-22 at 1 24 54 PM

File: vpc-native/main.tf [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
-  subnetwork         = var.nodes_subnetwork_name
-  monitoring_service = var.monitoring_service
-  logging_service    = var.logging_service
+  name                     = var.name
+  location                 = var.region
+  min_master_version       = var.kubernetes_version
+  network                  = var.network_name
+  subnetwork               = var.nodes_subnetwork_name
+  monitoring_service       = var.monitoring_service
+  logging_service          = var.logging_service
```
Comment (sudermanjr) [inline vpc-native/main.tf:8]: I think a `terraform fmt` will fix this

**Review by sudermanjr — APPROVED** [review]
Tested on a cluster I have and it works. Thanks!

File: vpc-native/main.tf (hunk 2 above)
Comment (transient1) [reply vpc-native/main.tf]: Ah, I see. Missed it. Was looking at this reference:

```The vertical_pod_autoscaling block supports:

    enabled (Required) - Enables vertical pod autoscaling
```

**Pull request merged**